src/app/components/Connect/ConnectedSmall.tsx

```diff
@@ -22,87 +20,107 @@ interface ConnectedSmallProps {
 
 export const ConnectedSmall: React.FC<ConnectedSmallProps> = ({
   loading = false,
-  connected,
   address,
   btcWalletBalanceSat,
   onDisconnect,
 }) => {
   const { ordinalsExcluded, includeOrdinals, excludeOrdinals } = useAppState();
+  const { connected } = useWalletConnect();
+  const btc = useBTCWallet();
+  const bbn = useCosmosWallet();
 
   const [showMenu, setShowMenu] = useState(false);
   const handleClickOutside = () => {
     setShowMenu(false);
   };
 
+  useEffect(() => {
+    const action = async () => {
+      console.log({ btc, network: await btc.getNetwork(), bbn });
+    };
+    action();
+  }, [bbn, btc]);
+
   const ref = useRef(null);
   useOnClickOutside(ref, handleClickOutside);
 
   const { coinName } = getNetworkConfig();
```

Comment:
You can call it once outside of render function

Reply:
This is not used so I will remove it for now.